Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/Coupled/Chiller.mo

```diff
@@ -0,0 +1,284 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.Coupled;
+model Chiller "Motor coupled chiller"
+
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    m1_flow_nominal = QCon_flow_nominal/cp1_default/dTCon_nominal,
+    m2_flow_nominal = QEva_flow_nominal/cp2_default/dTEva_nominal);
+  extends Buildings.Electrical.Interfaces.PartialOnePort(
+    redeclare package PhaseSystem =
+        Buildings.Electrical.PhaseSystems.OnePhase,
+    redeclare replaceable Interfaces.Terminal_n terminal);
+
+  //Chiller parameters
+  parameter Modelica.Units.SI.HeatFlowRate QEva_flow_nominal(max=0) = -P_nominal * COP_nominal
+    "Nominal cooling heat flow rate (QEva_flow_nominal < 0)"
```

Comment:
in comment, best would be not showing the parameter/variable name. So the comment should be `Nominal cooling heat flow rate (negative)`. Also, add
```
initial equation 
  assert(QEva_flow_nominal < 0, "Parameter QEva_flow_nominal must be negative.");
```

Reply:
Thank you for your suggestions! Originally I just referred to the chiller model in MBL (https://github.com/lbl-srg/modelica-buildings/blob/8ab33d18a40dec150af2d10335f31d7cd002db12/Buildings/Fluid/Chillers/BaseClasses/Carnot.mo#L11), It is definitely more helpful to understand with a clear description. I have revised the comment and added initial equation.